index.js

```diff
@@ -45,6 +45,30 @@ module.exports = {
   rules: {
     // open a PR per rule change
 
+    // Simplifies code and reduces bugs
+    // https://eslint.org/docs/rules/no-lonely-if
+    'no-lonely-if': ['error'],
+
+    // Reduces careless typo mistakes
+    // https://eslint.org/docs/rules/no-return-assign
+    'no-return-assign': ['error'],
+
+    // Spreading is better than complex Object.assign
+    // https://eslint.org/docs/rules/prefer-object-spread
+    'prefer-object-spread': ['error'],
+
+    // Not all arrow functions need to return
+    // https://eslint.org/docs/rules/consistent-return
+    'consistent-return': 'off',
+
+    // This catches typos that end up being bitwise operator
+    // However, some operators are common and should be allowed
+    // https://eslint.org/docs/rules/no-bitwise
+    'no-bitwise': ['error', { allow: ['~'] }],
```

Comment:
Is `~` really common? I wouldn't call any bitwise operators common

Reply:
~~ is covered by this rule. You can't allow ~~ only ~. I use ~~ a lot.

Reply:
I haven't seen bitwise used in any Modules far as I recall. Do we really need an opinion on bitwise?

Reply:
I believe the idea is that this catches accidental bitwise, where you meant to do && not & or || not |. To be honest, with modern IDEs, this almost never happens to me anymore.

That said, I'm fine with enabling this rule again as long as we have allow `~` so I can continue using `~~` like a boss.

Reply:
does using `~~` lead to more understandable code ? is the use case common enough to justify the exception ? how many engineers are likely to come across `~~somevar` and understand it ? and are the gains worth the potential confusion ?

Reply:
I'm not going to die on this hill. If you don't want this exception, we can take it out.

Reply:
I wrote a little [article](https://medium.com/@s.kutyepov/how-a-k-a-double-bitwise-not-operator-in-javascript-can-help-boost-performance-of-your-app-3d973078ff42) about `~~` not that long ago. It is definitely cool in some cases but should be used sparingly and with caution. 
Specifically:
>  ~~ will not work as a replacement of Math.floor()on an integer greater than 2147483647 or less than -2147483648 (max value for signed 32 bit integer).

```
~~(Math.pow(2, 31) + 1.5);
// Expected: 2147483649
// Result: -2147483647
```